lib/internal/process.js

```diff
@@ -118,18 +118,19 @@ function setupConfig(_source) {
     return value;
   });
   const processConfig = process.binding('config');
+  // XXX we don't have Intl currently
```

Comment:
I just fixed the ICU setup in 1f59e06fe1d27689aa5bcc856a05a4812ccfad5e.  Now you should be able to remove this hunk without any side effects.  If this can't be done, please file a new issue.